modules/dr-recover-lost-control-plane-hosts.adoc

```diff
@@ -0,0 +1,278 @@
+----
+[core@ip-10-0-156-255 ~]$ oc login https://localhost:6443
+Authentication required for https://localhost:6443 (openshift)
+Username: kubeadmin
+Password:
+Login successful.
+----
+
+... Obtain a local copy of the pull secret used to install the cluster.
++
+----
+[core@ip-10-0-156-255 ~]$ oc get secret pull-secret -n openshift-config -o jsonpath={.data.'\.dockerconfigjson'} | base64 -d > config.json
+----
+
+... Export two environment variables that are required by the `etcd-member-recover.sh` script.
++
+----
+[core@ip-10-0-156-255 ~]$ export SETUP_ETCD_ENVIRONMENT=$(oc adm release info --image-for setup-etcd-environment --registry-config=./config.json)
+----
++
+----
+[core@ip-10-0-156-255 ~]$ export KUBE_CLIENT_AGENT=$(oc adm release info --image-for kube-client-agent --registry-config=./config.json)
+----
+
+... Run the `etcd-member-recover.sh` script.
++
+Be sure to pass in the `-E` flag to `sudo` so that environment variables are properly passed to the script.
++
+----
+[core@ip-10-0-156-255 ~]$ sudo -E /usr/local/bin/etcd-member-recover.sh 10.0.143.125 <1>
+Downloading etcdctl binary..
+etcdctl version: 3.3.10
+API version: 3.3
+etcd-member.yaml found in ./assets/backup/
+etcd.conf backup upready exists ./assets/backup/etcd.conf
+Trying to backup etcd client certs..
+etcd client certs already backed up and available ./assets/backup/
+Stopping etcd..
+Waiting for etcd-member to stop
+etcd data-dir backup found ./assets/backup/etcd..
+etcd TLS certificate backups found in ./assets/backup..
+Removing etcd certs..
+Populating template /usr/local/share/openshift-recovery/template/etcd-generate-certs.yaml.template
+Populating template ./assets/tmp/etcd-generate-certs.stage1
+Populating template ./assets/tmp/etcd-generate-certs.stage2
+Starting etcd client cert recovery agent..
+Waiting for certs to generate..
+Waiting for certs to generate..
+Waiting for certs to generate..
+Waiting for certs to generate..
+Stopping cert recover..
+Waiting for generate-certs to stop
+Patching etcd-member manifest..
+Updating etcd membership..
+Member 249a4b9a790b3719 added to cluster 807ae3bffc8d69ca
+
+ETCD_NAME="etcd-member-ip-10-0-156-255.ec2.internal"
+ETCD_INITIAL_CLUSTER="etcd-member-ip-10-0-143-125.ec2.internal=https://etcd-0.clustername.devcluster.openshift.com:2380,etcd-member-ip-10-0-156-255.ec2.internal=https://etcd-1.clustername.devcluster.openshift.com:2380"
+ETCD_INITIAL_ADVERTISE_PEER_URLS="https://etcd-1.clustername.devcluster.openshift.com:2380"
+ETCD_INITIAL_CLUSTER_STATE="existing"
+Starting etcd..
+----
+<1> The IP address of the original master, where the signer server is running.
+
+... Verify that the new master host has been added to the etcd member list.
+
+.... Access the original master and connect to the running etcd container.
++
+----
+[core@ip-10-0-143-125 ~] id=$(sudo crictl ps --name etcd-member | awk 'FNR==2{ print $1}') && sudo crictl exec -it $id /bin/sh
+----
+
+.... Export variables needed for connecting to etcd.
++
```

Comment:
I know the prompt changes but should we remind the user that this is run inside the container?